worlds/cv64/rom.py

```diff
@@ -0,0 +1,907 @@
+import Utils
+
+from worlds.Files import APDeltaPatch
+
+import hashlib
+import os
+import pkgutil
+
+from . import lzkn64
+from .data import patches
+from .stages import get_stage_info
+from .text import cv64_string_to_bytes, cv64_text_truncate, cv64_text_wrap
+from .aesthetics import renon_item_dialogue, get_item_text_color
+from .options import CV64Options
+from .locations import get_location_info
+from settings import get_settings
+
+CV64US10HASH = '1cc5cf3b4d29d8c3ade957648b529dc1'
+ROM_PLAYER_LIMIT = 65535
+
+warp_map_offsets = [0xADF67, 0xADF77, 0xADF87, 0xADF97, 0xADFA7, 0xADFBB, 0xADFCB, 0xADFDF]
+
+
+class LocalRom(object):
```

Comment:
I think the `(object)` is redundant. A remnant of Python2 maybe?

Reply:
This still exists in DKC3, funnily enough, where I copied much of this from initially.